test/mailers/contact_mailer_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class ContactMailerTest < ActionMailer::TestCase
+  test "contact_us" do
+    contact = Contact.new(name: "person",
```

Comment:
Since you have a `test_first_contact` in the contacts fixture, it would probably be better to use that. That way the test won't have the string values of the email and message copy pasted.